pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -117,7 +118,12 @@ stdenv.mkDerivation rec {
     ] ++ stdenv.lib.optional (stdenv.hostPlatform.libc == "musl") "NO_AFFINITY=1"
     ++ mapAttrsToList (var: val: var + "=" + val) config;
 
-  patches = []; # TODO: Remove on next mass-rebuild
+    patches = [
```

Comment:
Why? Please add a comment.

Reply:
Thats the openblas backport I mentioned in the PR description, #43234